base/special/exp.jl

```diff
@@ -460,7 +460,7 @@ function expm1(x::Float32)
     end
     x = Float64(x)
     N_float = round(x*Ln2INV(Float64))
-    N = unsafe_trunc(UInt64, N_float)
+    N = unsafe_trunc(Int64, N_float)
```

Comment:
I imagine MAX_EXP is much smaller than the maximum integer here? If so LGTM

Reply:
yeah. `N` will end up as the exponent for the float so it will be in [-128,128]